artifact/src/main/java/groupgroup/Configuration.java

```diff
@@ -20,6 +20,7 @@
     private static final String REGULATOR_TCR_SCALING_FACTOR = "regulator.TCRScalingFactor";
 	private static final String REGULATOR_INITIAL_TCR = "regulator.initialTCR";
 	private static final String REGULATOR_TCR_LIVE_UPDATE = "regulator.TCRLiveUpdate";
+	private static final String REGULATOR_OLD_JOB_WEIGHT_FACTOR = "regulator.oldJobWeightFactor";

```

Comment:
This variable isn't used anymore, will we need it in the future? Used for configuring an aging average.